cop-16-random-person-creation

nirbelelti opened the pull request:
for testing
```
def main():
    
    print(Person.create_passengers(["Lyngby","København H"], {"start": datetime.now(), "end": datetime.now()+timedelta(hours=9)}, 100))
    return
```

nirbelelti committed:
Person class and statistic class

nirbelelti committed:
Add clas add passengers generator

KuroHaka committed:
Merge branch 'main' into cop-16-random-person-creation

KuroHaka commented on train_simulation/Passengers_generator.py:
```diff
@@ -0,0 +1,51 @@
+class Passengers_generator:
+    passengers = {}
+
+    def __init__(self, low_load, mid_load, high_load, interval, current_load, station):
```
https://linear.app/copenhagen-railway/issue/COP-16/random-person-creation
check new parameters

nirbelelti committed:
remove old class

nirbelelti committed:
Merge remote-tracking branch 'origin/cop-16-random-person-creation' into cop-16-random-person-creation

KuroHaka commented on train_simulation/Person.py:
```diff
@@ -1,3 +1,48 @@
+        return self.end_time - self.time
+
+    @staticmethod
+    def create_passengers(critical_stations, time, n_passengers):
```
Not working, can't use this method, call me

nirbelelti replied:
tested several times, Please try
`critical_stations = ['Valby','Allerød','Holte']
passengers= Person.create_passengers(critical_stations, datetime.now(), 100)`

work fine at my side

KuroHaka replied:
can't use it in main class, fix imports since you are opening a json file from this class. Another option is change station as a parameter

nirbelelti committed:
Add ability to except rage of date and generate random date
Write the generated passengers into a json file
Generate list of passenger train ride start and end stations

KuroHaka committed:
done

KuroHaka committed:
merge with main

KuroHaka committed:
Merge branch 'main' into cop-16-random-person-creation
